src/render/traverse.js

```diff
@@ -0,0 +1,154 @@
+const { isArray, isFunction, isString } = require("lodash");
+
+const { getChildContext, getContext } = require("./context");
+const { syncSetState } = require("./state");
+const { getCachedSequence, sequence } = require("../sequence");
+const { toDashCase, htmlStringEscape, hasOwn } = require("./util");
+
+
+function renderAttrs (attrs) {
+  const attrString = [];
+
+  for (const attrKey in attrs) {
+    if (
+      hasOwn(attrs, attrKey) &&
+      attrKey !== "children"
+    ) {
+      const attrVal = attrs[attrKey];
+
+      if (!attrVal || isFunction(attrVal)) { continue; }
+
+      attrString.push(` ${toDashCase(attrKey)}${attrVal ? "" : "${attrVal}"}`);
+    }
+  }
+
+  return attrString.join("");
```

Comment:
_Really..._ I thought the opposite was true!  In that case, I'd be very interested in switching it out.  Thanks for the tip!